webrender/src/renderer.rs

```diff
@@ -4066,7 +4121,7 @@ impl Renderer {
                             alpha.max_size.height as f32,
                             ORTHO_NEAR_PLANE,
                             ORTHO_FAR_PLANE,
-                        );
+                        ).post_scale(1.0, -1.0, 1.0);
```

Comment:
let's put a big fat comment here saying why the scaling is needed. Or better - modify the parameters to `ortho` so that scaling is not needed :)